camayoc/tests/qcs/cli/test_credentials.py

```diff
@@ -26,16 +26,17 @@
 
 def generate_show_output(data):
     """Generate a regex pattern with the data for a qpc cred show output."""
+    cred_type = data.get('cred_type', 'network')
     output = '{\r\n'
-    output += '    "become_method": "{}",\r\n'.format(
-        data.get('become_method', 'sudo'))
-    if 'become_password' in data:
-        output += '    "become_password": "{}",\r\n'.format(
-            data['become_password'])
-    output += '    "become_user": "{}",\r\n'.format(
-        data.get('become_user', 'root'))
-    output += '    "cred_type": "{}",\r\n'.format(
-        data.get('cred_type', 'network'))
+    if cred_type == 'network':
+        output += '    "become_method": "{}",\r\n'.format(
+            data.get('become_method', 'sudo'))
```

Comment:
so it looks like here you allways have a become method. Is that your intention?

Reply:
become_method has the default of sudo, that is why whenever a network credential is created the value will be presented as sudo even if no become_method is passed during the creation.